README.md

```diff
@@ -1,129 +1,199 @@
+var language = "JavaScript";
+//=> undefined
 
-## Multi-line Variable Assignment
+language;
+//=> "JavaScript"
+```
 
-Let's say I needed to declare and define multiple variables. It feels like a lot to have to repeat `var` over and over again. JavaScript allows us to do multi-line variable assignment to alleviate this pain. Every variable must be separated with a comma, and the entire line must end with a semicolon.
+This is bad! There's no reason to declare a variable twice, and it's usually a mistake by a developer unaware that the variable had already been declared.
 
-Let's condense the below code into one line:
+### `let`: the new `var`
+ES2015 introduced two new ways to create variables: `let` and `const`. Both solve all of `var`'s scope issues, which, again, we'll cover in the lesson on scope in JavaScript. Both also throw an error if you try to declare the same variable a second time:
+```js
+let pi = 3.14159;
+//=> undefined
 
-```javascript
-var a = 5
-var b = 2
-var c = 3
-var d = 'hello'
-var e = 'goodbye'
+let pi = "the ratio between a circle's circumference and diameter";
+//=> Uncaught SyntaxError: Identifier 'pi' has already been declared
 ```
 
-The above is equivalent to:
-
-```javascript
-var a = 5,
-    b = 2,
-    c = 3,
-    d = 'hello',
-    e = 'goodbye';
-```
+Just like with `var`, we can still reassign a variable declared with `let`:
+```js
+let pi = 3.14159;
+//=> undefined
 
-which can be converted to:
+pi = "the ratio between a circle's circumference and diameter";
+//=> "the ratio between a circle's circumference and diameter"
 
-```javascript
-var a = 5, b = 2, c = 3, d = 'hello', e = 'goodbye'
+typeof pi;
+//=> "string"
 ```
 
-Try typing each variable in the console. You should see the appropriate values returned for each one.
+### "Imma `let` you finish, but `const` is the best way to declare variables of all time"
+![Kanye](https://user-images.githubusercontent.com/17556281/28677639-21dd517e-72bc-11e7-9759-f8f2f69d34e0.gif)
+
+Using `let` instead of `var` will help you avoid silly errors like declaring the same variable at two different places within your code, but there's an even better option to use as your default: `const`.
 
-Personally, we're of the opinion that it's best to declare each variable with its own `var` keyword. The comma fanciness saves a little bit of typing, but at the expense of nonstandard indentation (you're using two spaces, right?) and can introduce a bit of weirdness if we're doing anything other than simple assignment.
+Declaring a variable with the `const` reserved word means that not only can it not be redeclared but it also ***cannot be reassigned***. This is a good thing for two reasons:
+1. When we assign a primitive value (a number, string, boolean, etc. — anything except an object) to a variable declared with `const`, we know that variable will _always_ contain the same value.
```

Comment:
@jmburges 

I can probably remove "primitive" and just make the distinction "number, string, boolean, symbol, undefined, null" vs. "object"